os-autoinst-obs-auto-submit

```diff
@@ -232,6 +232,7 @@ is_staging_specified() {
 
 submit() {
     # submit from staging project if specified
+    is_staging_specified && src_project=$staging_project
```

Comment:
```suggestion
    is_staging_specified && src_project=$staging_project must_cleanup_staging=1
```

And remove the assignment of `must_cleanup_staging` in `is_staging_specified` instead. (A function called `is_…` should not have side-effects like changing variables.)